templates/index.html

```diff
@@ -29,10 +29,10 @@ <h1>Signup</h1>
     <br>
 
     <label>Email
-        <input type="text" name="verify_password" value="{{verify_password}}" />
+        <input type="text" name="email" value="{{email}}" />
         </label>
-        {% if verify_error %}
-        <span class='error'>{{ verify_error }}</span>
+        {% if email_error %}
+        <span class='error'>{{ email_error }}</span>
```

Comment:
You actually had email set to the same thing as the "Verify password" field, so I changed it to say email, and included `email_error`